querybook/webapp/components/StatementResultTable/StatementResultTable.tsx

```diff
@@ -109,6 +121,23 @@ export const StatementResultTable: React.FunctionComponent<{
         filterConditionByColumn,
     } = useFilterCell(rows);
 
+    const [isDisplayFull, setIsDisplayFull, prevValue] = useSavePrevValue();
+
+    const visibilityToggle = () => {
+        if (!isDisplayFull) {
+            prevValue.current = { ...expandedColumn };
+            toggleExpandedColumn(
```

Comment:
no need to use toggleExpandedColumn, use setExpandedColumn and isDisplayFull to just set all columns to true or false